src/test/java/testPages/SignInPageTest.java

```diff
@@ -0,0 +1,38 @@
+package testPages;
+
+import base.BaseTest;
+import io.qameta.allure.Description;
+import listeners.TestListener;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import pages.HomePage;
+
+@Listeners(TestListener.class)
+public class SignInPageTest extends BaseTest {
+
+    private static final String EXPECTED_SIGN_IN_MESSAGE = "Успешно";
+    private static final String CORRECT_LOGIN = "kseniya123@mail.ru";
+    private static final String EMPTY_LOGIN = "";
+    private static final String EMPTY_PASSWORD = "";
+
+    @Test(priority = 1, description = "[TC#5] - Check authorization with incorrect password", dataProvider = "incorrectPasswords")
+    @Description("Test case description: check authorization with incorrect password")
+    public void checkAuthorizationWithIncorrectPassword(String password) {
+        String actualSignInMessage = new HomePage().openSignInForm().enterLoginAndPassword(CORRECT_LOGIN, password).getSignInMessage();
```

Comment:
What's the point in data provider if there is only one data value there? in what cases do we use a data provider?
And i thought we agreed to add MULTIPLE data combinations for log in test with incorrect data.

Reply:
And what if you want to try combination with CORRECT username but incorrect password? but it's already specifies inside the test